apps/web/src/services/dashboards/widgets/severity-status-by-service/SeverityStatusByServiceWidget.vue

```diff
@@ -16,7 +16,7 @@
                 </div>
             </div>
             <div class="legend-wrapper">
-                <div v-for="status in Object.values(SEVERITY_STATUS)"
+                <div v-for="status in Object.values(SEVERITY_STATUS_MAP)"
```

Comment:
nitpick: This is good but it executes `Object.values(SEVERITY_STATUS_MAP) `everytime.. for more efficient way, you can change it to use just a constant value. :)